styles.css

```diff
@@ -0,0 +1,91 @@
+
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: auto;
+  font-family: 'Poppins', sans-serif !important;
```

Comment:
- [ ] [ OPTIONAL ] To achieve the best styles on your page, it is recommended that you avoid using `!important` since some of your styles might fail to work correctly when your app gets bigger. Please consider implementing your styles in a way that they work correctly without having to use `!important`.